src/reputation/views/withdrawal_view.py

```diff
@@ -28,15 +28,19 @@
 from reputation.models import Withdrawal
 from reputation.permissions import AllowWithdrawalIfNotSuspecious
 from reputation.serializers import WithdrawalSerializer
-from researchhub.settings import ETHERSCAN_API_KEY, WEB3_PROVIDER_URL, WEB3_RSC_ADDRESS
+from researchhub.settings import (
+    BASESCAN_API_KEY,
+    ETHERSCAN_API_KEY,
+    WEB3_PROVIDER_URL,
+    WEB3_RSC_ADDRESS,
+)
```

Comment:
Import `settings` not the settings themselves.

Reply:
I have an upcoming PR that is based on this and fixes it.